src/witan/send/adroddiad/valid_states.clj

```diff
@@ -20,22 +20,24 @@
     (ay/key-stage-3 y) (apply max ay/key-stage-3)
     (ay/key-stage-4 y) (apply max ay/key-stage-4)
     (ay/key-stage-5 y) (apply max ay/key-stage-5)
-    (ay/ncy-15+ y) (apply max ay/ncy-15+)))
+    (ay/ncy-15+ y) (apply max ay/ncy-15+)
+    (< (apply max ay/ncy-15+) y) (apply max ay/ncy-15+)))
 
 (defn candidate-valid-states [census-data]
   (let [needs (->> (into (sorted-set) (:need census-data))
                    (interpose \,)
                    (apply str))
         settings (->> (into (sorted-set) (:setting census-data))
                       (interpose \,)
-                      (apply str))]
+                      (apply str))
+        ncys (into (sorted-set) (:academic-year census-data))]
     (-> census-data
         (tc/select-columns [:setting :academic-year])
         (tc/group-by (fn [row] (-> (:setting row)
                                    (clojure.string/split #"_")
                                    first)))
-        (tc/aggregate {:min-academic-year #(keystage-min-ay (reduce min (:academic-year %)))
-                       :max-academic-year #(keystage-max-ay (reduce max (:academic-year %)))})
+        (tc/aggregate {:min-academic-year #(max (apply min ncys) (keystage-min-ay (reduce min (:academic-year %))))
+                       :max-academic-year #(min (apply max ncys) (keystage-max-ay (reduce max (:academic-year %))))})
```

Comment:
This one bugs me more as I'm not sure why, other than specific instruction we'd want to cut out a max ncy from a keystage if we didn't have a specific reason to. I do know we sometimes get spurious outliers, esp for EYS settings, but I think we should just change it manually after review.